server/src/main/java/org/opensearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -135,7 +135,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeEnum(level);
         }
         if (out.getVersion().onOrAfter(Version.V_2_6_0)) {
-            out.writeBoolean(ensureNodeCommissioned);
+            out.writeBoolean(ensureNodeWeighedIn);
```

Comment:
Will this cause BWC failures? If yes we can retain both flags and deprecate the former

Reply:
Since `2.6.0` is not released yet, do you this BWC is required? I am looking out for test failures in gradle build, will that be sufficient?